nautobot_golden_config/jobs.py

```diff
@@ -171,14 +178,20 @@ def run(self, data, commit):
         """Run config backup process."""
         now = datetime.now()
         LOGGER.debug("Pull Backup config repo.")
-        backup_repo = git_wrapper(self, GoldenConfigSetting.objects.first().backup_repository, "backup")
+        golden_settings = GoldenConfigSetting.objects.first()
+
+        # Instantiate a GitRepo object for each GitRepository in GoldenConfigSettings.
+        backup_repos = [git_wrapper(self, repo, "backup") for repo in golden_settings.backup_repository.all()]
+        LOGGER.debug("Starting backup jobs to the following repos: %s", backup_repos)
 
         LOGGER.debug("Run nornir play.")
-        config_backup(self, data, backup_repo.path)
+        config_backup(self, data, backup_repos)
 
-        LOGGER.debug("Pull Backup config repo.")
-        backup_repo.commit_with_added(f"BACKUP JOB {now}")
-        backup_repo.push()
+        LOGGER.debug("Pushing Backup config repo.")
```

Comment:
Should keep context of which repo is in the debug message, and keep this in the loop as the other similar instance is.